isis/src/mro/apps/himos/himos.cpp

```diff
@@ -286,7 +286,7 @@ namespace Isis {
   } // end of isis main
 
   //Function to compare label - CompareLabels
-  void CompareLabels(Pvl &pmatch, Pvl &pcomp) {
+  static void CompareLabels(Pvl &pmatch, Pvl &pcomp) {
```

Comment:
I'm pretty sure you only need it on the forward declaration but if the compiler doesn't complain it's whatever.